contracts/savings/Savings.sol

```diff
@@ -24,13 +24,19 @@ contract Savings is BaseSavings {
     /// @notice Whether the contract is paused or not
     uint8 public paused;
 
+    /// @notice Maximum inflation rate
+    /// @dev Note that `rate` can still be greater than `maxRate` if this `maxRate` is reduced by governance
```

Comment:
to fix that we can always set the rate to a lower amount when setting maxRate in the governor function